README.md

```diff
@@ -50,28 +80,58 @@ just m1-da-light-node native build.setup.test.local
 just monza-full-node native build.setup.test.local
 ```
 
-## Run with Docker Compose
-When running with `docker compose` specif your revision in a file `.env` at the root of the project. The file should look like this:
+## Run a Movement Node with Docker Compose
+1. Make sure you have installed the `just` command on your system. If not check the 
+"Prerequisites" section of this repo.
 
+2. When running with `docker compose` specify your revision in a file `.env` at the root of
+the project. The file should look like this:
 ```bash
+# /path/to/movement/.env
 CONTAINER_REV=0fe2a4f28820c04ca0db07cdd44cafc98b792f3f
 ```
 
+We recommend to use the latest commit of the "main" branch:
+```bash
+GIT_ROOT=$(git rev-parse --show-toplevel)
+MOVEMENT_ENV_FILE="${GIT_ROOT}/.env"
+[[ -n "${GIT_ROOT}" ]] \
+  && echo  "CONTAINER_REV=$(git rev-parse HEAD)" > "${MOVEMENT_ENV_FILE}"
+echo "INFO: movement version is $(cat ${MOVEMENT_ENV_FILE})"
+```
+
 ### `suzuka-full-node`
 
 - **Features**:
     - `setup`: Run setup for new `suzuka-full-node` network with single node.
     - `local`: Run a local Celesta Data Availability service.
 
-**Note:** Currently, both `setup` and `local` must be used. We only support running the `suzuka-full-node` with a local Celesta Data Availability service via Docker Compose.
+**Note:** Currently, both `setup` and `local` must be used. 
+We only support running the `suzuka-full-node` with a local Celesta Data Availability 
+service via Docker Compose.
 
 ```bash
 # example setup with local
 just suzuka-full-node docker-compose setup.local
 ```
+Under the hood, `just` runs
+```bash
+# working directory = GIT_ROOT
+GIT_ROOT=$(git rev-parse --show-toplevel)
+docker compose --env-file .env \
+               --file docker/compose/suzuka-full-node/docker-compose.yml \
+               --file docker/compose/suzuka-full-node/docker-compose.setup.yml \
+               --file docker/compose/suzuka-full-node/docker-compose.local.yml \
+               up
+```
 
 **Note:** if you want to recreate the network, but not rely on the just target above, please read through the scripts to identify the correct `docker-compose` files to run.
 
+**Note** For attesters in order to receive rewards you need to launch the node in 
+"attester mode". To do this you will need to provide a private key at runtime. 
```

Comment:
I use MLA for quotes, so the period would go on the inside here. Probably need to decide as a team what standard we are using. For this in particular, I might just do `ATTESTER` mode.

Reply:
Can you please add a suggestion. I don't understand what you want me to do here

Reply:
Change "attester mode" to "`Attester` mode"--without any surrounding quotes.